pkg/component/extensions/backupentry/backupentry.go

```diff
@@ -109,6 +109,7 @@ func (b *backupEntry) deploy(ctx context.Context, operation string) (extensionsv
 	_, err := controllerutils.GetAndCreateOrMergePatch(ctx, b.client, b.backupEntry, func() error {
 		metav1.SetMetaDataAnnotation(&b.backupEntry.ObjectMeta, v1beta1constants.GardenerOperation, operation)
 		metav1.SetMetaDataAnnotation(&b.backupEntry.ObjectMeta, v1beta1constants.GardenerTimestamp, b.clock.Now().UTC().Format(time.RFC3339Nano))
+		metav1.SetMetaDataLabel(&b.backupEntry.ObjectMeta, v1beta1constants.LabelExtensionProviderTypePrefix+b.values.Type, "true")
```

Comment:
Shouldn't the label be documented as part of the extensibility contract so that extension developers know about it and make use of it when they create webhooks in the Seed for these objects?